### PR TITLE
[CI] use obltGitHubComments

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Use https://github.com/elastic/apm-pipeline-library/blob/master/vars/obltGitHubComments.txt

## Why is it important?

Avoid hardcoded strings and easy to normalise the GitHub comments in all the oblt projects

You can also see in the GitHub comment that there is a specific section for **some** of those GitHub comments called `🤖 GitHub comments`

### CODE REVIEW

**Review by kuisathaverat — APPROVED** [review]

File: .ci/Jenkinsfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?(?:benchmark\\W+)?tests(?:\\W+please)?.*')
+    issueCommentTrigger("(${obltGitHubComments()}|^run benchmark tests)")
```
Comment (cachedout) [inline .ci/Jenkinsfile:42]: Is it a problem that the world `please` was removed from the end or will that still be accepted?

**Comment by v1v** [comment]
run benchmark tests

File: .ci/Jenkinsfile (hunk 1 above)
Comment (v1v) [reply .ci/Jenkinsfile:42]: It's accepted, see https://github.com/elastic/apm-agent-ruby/pull/1164#issuecomment-924832635 that triggered https://apm-ci.elastic.co/job/apm-agent-ruby/job/apm-agent-ruby-mbp/job/PR-1164/3/

**Pull request merged**